packages/linter/tests/local/ImagesHaveAltText-1/source.html

```diff
@@ -0,0 +1,290 @@
+            }
+        </style>
+        <noscript>
+            <style amp-boilerplate>
+                body {
+                    -webkit-animation: none;
+                    -moz-animation: none;
+                    -ms-animation: none;
+                    animation: none
+                }
+            </style>
+        </noscript>
+        <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+        <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+        <title>Through the fence 2020</title>
+        <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+        <link rel="shortcut icon" href="https://app.makestories.io/images/favicon.png" type="image/x-icon">
+        <script type="application/ld+json">
+            {"@context":"https://schema.org","@type":"event","mainEntityOfPage":{"@type":"WebPage","@id":"https://onl.st/linter-tester"},"name":"Dee Brown","description":"Come gather at the 2020 Through the Fence Festival","image":{"@type":"ImageObject","url":"https://images.unsplash.com/photo-1533794318766-897f4d50cb39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjI1MTY2fQ&q=85&w=2160","width":1280,"height":1920},"location":{"@type":"Place","name":"","address":{"@type":"PostalAddress","streetAddress":"","addressLocality":"","addressRegion":"","postalCode":"","addressCountry":""}},"startDate":"05/01/2020T5:54:33","endDate":"05/02/2020T4:57:33","eventStatus":"https://schema.org/EventScheduled","eventAttendanceMode":"http://schema.org/OfflineEventAttendanceMode","performer":{"@type":"Person","name":""}}
+        </script>
+        <meta property="og:url" content="https://onl.st/linter-tester">
+        <meta property="title" content="Through the fence 2020">
+        <meta property="og:title" content="Through the fence 2020">
+        <meta property="twitter:title" content="Through the fence 2020">
+        <meta property="keywords" name="keywords" content="Through the fence festival music fun ">
+        <meta property="description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="og:description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="twitter:description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="twitter:image" content="https://images.unsplash.com/photo-1533794318766-897f4d50cb39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjI1MTY2fQ&q=85&w=2160">
+        <meta property="og:image:url" content="https://images.unsplash.com/photo-1533794318766-897f4d50cb39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjI1MTY2fQ&q=85&w=2160">
+        <meta property="og:image:width" content="1280">
+        <meta property="og:image:height" content="1920">
+        <meta name="twitter:card" content="summary_large_image">
+        <link rel="canonical" href="https://onl.st/linter-tester">
+        <style amp-custom>
```

Comment:
Can you clean up this file and remove everything that is not relevant for the test (e.g. CSS)